skill-tree/src/main/java/com/RDS/skilltree/Endorsement/EndorsementModelFromJSON.java

```diff
@@ -0,0 +1,40 @@
+package com.RDS.skilltree.Endorsement;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.*;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+public class EndorsementModelFromJSON {
+    private UUID id;
+
+    @JsonProperty("user_id")
+    private UUID userID;
```

Comment:
please use upper case for first chars of every props

Reply:
One small confusion.. 
do you mean the json user_id should be User_id or the variable name should be UserId?
(Basically change needed on line 18 or 19?)